formats/protobuf/commonMain/src/kotlinx/serialization/protobuf/internal/ProtobufDecoding.kt

```diff
@@ -16,6 +16,10 @@ import kotlinx.serialization.modules.*
 import kotlinx.serialization.protobuf.*
 import kotlin.jvm.*
 
+private const val FILLED_OR_DEFAULT: Byte = 0
```

Comment:
I constantly think that this is a mistypo of 'FIELD'. Maybe we need a better name, e.g. PRESENT_OR DEFAULT